teachertool/src/services/loggingService.ts

```diff
@@ -0,0 +1,26 @@
+import { isLocal } from "../utils";
```

Comment:
Oh, this is something I forgot to change. It's not something that needs to be done in this PR, but Eric pointed out to me that we have this function in `pxt.util`, so we can technically remove it from the teachertool util.

https://github.com/microsoft/pxt/pull/9806#discussion_r1439775765